build.wake

```diff
@@ -92,6 +92,10 @@ tuple FirrtlCompilePlan =
   global SplitModules:     Boolean
   global VisibleFiles:     List Path
   global JavaOpts:         List String
+  global Main:             String
```

Comment:
Can we give this a more descriptive name, such as `MainClass`? I don't think it is obvious what this field contains to someone just reading the tuple definition, and this tuple is an important public API, so it is most important to make the tuple as self-documenting as possible.